dashbuilder-client/dashbuilder-navigation-client/src/main/java/org/dashbuilder/client/navigation/widget/NavComponentConfigModalView.css

```diff
@@ -0,0 +1,20 @@
+.navconfig-body {
```

Comment:
could you please append a prefix for the css? to avoid future conflicts

Reply:
Is there any existing convention for such prefix? I mean, I already consider "navconfig" a prefix. Would you mean we should add "kie", "uf" or whatever as a prefix?

Reply:
Yeah, just to prevent the class in future, something like uf-component or kie-component. Up to you.

Reply:
(but just a suggestion, not a blocker)